### PR TITLE
SelectBox - Getting Started: Add ASP.NET Core Overhaul

### PR DESCRIPTION
**Pull request by arman-boyakhchyan**

### CODE REVIEW

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/05 Create the SelectBox.md [hunk 1]
```diff
@@ -1 +1,93 @@
...
+        $("#selectBox").dxSelectBox({
+            // Configuration goes here
+        });
+    });
+
+    <!-- tab: index.html -->
+    <html>
+        <head>
+            <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+            <link rel="stylesheet" type="text/css" href="https://cdn3.devexpress.com/jslib/minor_26_2/css/dx.light.css" />
```
Comment (markallenramirez) [inline concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/05 Create the SelectBox.md]: switch to fluent theme

and just in case, `minor_26_2` will be replaced when displayed, yes?

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/05 Create the SelectBox.md [hunk 2]
```diff
@@ -1 +1,93 @@
...
+            <link rel="stylesheet" href="index.css">
```
Comment (markallenramirez) [inline concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/05 Create the SelectBox.md]:
```suggestion
            <link rel="stylesheet" href="index.css" />
```

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/15 Enable Search.md [hunk 3]
```diff
@@ -1,9 +1,57 @@
...
+To allow users to search through SelectBox values, set [searchEnabled](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/searchEnabled.md '/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#searchEnabled') to **true**:
+
+---
+
+##### jQuery
+
+    <!-- tab: index.js -->
+    $("#selectBox").dxSelectBox({
+        searchEnabled: true,
+        // ...
```
Comment (markallenramirez) [inline concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/15 Enable Search.md]: optional: remove here or add ellipsis in other platforms. main idea is consistency

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/26 Customize the Drop-Down Menu.md [hunk 4]
```diff
@@ -1,6 +1,67 @@
...
+        },
+        // ...
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-select-box
+        [dropDownOptions]="dropDownOptions">
+    </dx-select-box>
```
Comment (markallenramirez) [inline concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/26 Customize the Drop-Down Menu.md]:
```suggestion
    <dx-select-box>
        <dxo-select-box-drop-down-options
            [height]="150"
        ></dxo-select-box-drop-down-options>
    </dx-select-box>
```

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/26 Customize the Drop-Down Menu.md [hunk 5]
```diff
@@ -1,6 +1,67 @@
...
+        }
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxSelectBox
+            :drop-down-options="dropDownOptions"
+        />
```
Comment (markallenramirez) [inline concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/26 Customize the Drop-Down Menu.md]:
```suggestion
        <DxSelectBox>
            <DxDropDownOptions :height="150" />
        </DxSelectBox>
```

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/26 Customize the Drop-Down Menu.md [hunk 6]
```diff
@@ -1,6 +1,67 @@
...
+    
+    const dropDownOptions = {
+        height: 150,
+    };
+
+    export default function App() {
+        return (
+            <SelectBox
+                dropDownOptions={dropDownOptions}
+            />
```
Comment (markallenramirez) [inline concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/26 Customize the Drop-Down Menu.md]:
```suggestion
            <SelectBox>
                <DropDownOptions height={150} />
            </SelectBox>
```

**Review by markallenramirez — APPROVED** [review]

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/05 Create the SelectBox.md (hunk 1 above)
Comment (arman-boyakhchyan) [reply concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/05 Create the SelectBox.md]: Yes, `minor_26_2` is a placeholder that our site replaces automatically.

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/15 Enable Search.md [hunk 7]
```diff
@@ -1,9 +1,56 @@
...
+        return (
+            <SelectBox
+                searchEnabled={true}
+            />
+        );
+    }
+
+---
+
+For more information about the search capabilities of DevExtreme SelectBox, refer to the following demo:
```
Comment (dirkpieterse) [inline concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/15 Enable Search.md]:
```suggestion
For more information about DevExtreme SelectBox search capabilities, see the following demo:
```

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/20 Handle the Value Change Event.md [hunk 8]
```diff
@@ -1,3 +1,103 @@
-#include tutorials-editors-handle-value-change with {
-    output: "the IDs of the currently and previously selected items"
-}
+Use [onValueChanged](/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#onValueChanged) to specify a function that the component executes when users change the SelectBox value. This example implements the [notify](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_type_displayTime) utility method within this function:
```
Comment (dirkpieterse) [inline concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/20 Handle the Value Change Event.md]:
```suggestion
Use [onValueChanged](/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#onValueChanged) to specify a function that the component executes when users change the SelectBox value. The following example calls the [notify](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_type_displayTime) utility method within this function:
```

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/23 Add a Label.md [hunk 9]
```diff
@@ -1,3 +1,61 @@
-#include tutorials-editors-add-label with {
-    label: "Product"
-}
\ No newline at end of file
+To specify label text, define the [label](/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#label) property. To enable floating labels, set [labelMode](/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#labelMode) to *"floating"*. In floating mode, the label acts as a placeholder. When the editor is focused, the label shifts above the input field.
```
Comment (dirkpieterse) [inline concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/23 Add a Label.md]:
```suggestion
Set the [label](/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#label) property to specify label text. To enable floating labels, set [labelMode](/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#labelMode) to *"floating"*. In floating mode, the label acts as a placeholder and moves above the input field when the editor receives focus.
```

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/25 Group Data.md [hunk 10]
```diff
@@ -1,3 +1,120 @@
...
+            <SelectBox
+                dataSource={dataSource}
+                grouped={true}
+            />
+        );   
+    }
+    
+---
+
+SelectBox can also display grouped data using nested data sources with only one nesting level. Each object in a nested data source contains a **key** field and an **items** array. For more information, refer to the following guide: [SelectBox - Grouping in the Data Source](/Documentation/Guide/UI_Components/SelectBox/Grouping/In_the_Data_Source/).
```
Comment (dirkpieterse) [inline concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/25 Group Data.md]:
```suggestion
SelectBox can also display grouped data from a nested data source with one nesting level. Each object in the data source contains a key field and an items array. For more information, refer to the following guide: [SelectBox - Grouping in the Data Source](/Documentation/Guide/UI_Components/SelectBox/Grouping/In_the_Data_Source/).
```
